packages/core/src/stories/Installation/javascript.stories.ts

```diff
@@ -0,0 +1,103 @@
+import type { Meta } from '@storybook/html';
+import hljs from 'highlight.js';
+import '../../../.storybook/tegel.syntax.highlighter.css';
+
+const meta: Meta = {
+  title: 'Intro/Installation',
+  parameters: {
+    layout: 'fullscreen',
+    options: {
+      showPanel: false,
+      showToolbar: false,
+    },
+  },
+};
+
+hljs.highlightAll();
+
+export default meta;
+export const Javascript = {
+  render: () => `
+    <style>
+    article {
+        box-sizing: border-box;
+        max-width: 688px;
+
+        padding: 32px;
+        margin: auto;
+        >* {
+            margin-bottom: 72px;
+        }
+    }
+
+    code {
+        border-radius: 4px;
+    }
+</style>
+<article class="tds-u-p2 tds-body-01">
+    <section>
+        <p class="tds-body-01"><strong>Published: 2023-10-11</strong></p>
+        <h1 class="tds-headline-02">Installing Tegel</h1>
+        <p>This is a getting started guide aimed at developers that want to install and use the @scania/tegel library (TDS).
+            This library consists of web components built using Stencil, which means that they are framework agnostic and follow
```

Comment:
Saw that we sometimes refer to web components as "Web Components". Dunno what is correct, as I have seen both searching the internet. But we should maybe stick with one, and then use that and not mix. I have no preference on which one though.

Reply:
Agree! I updated the figma design for this task:https://tegel.atlassian.net/browse/CDEP-2525 to use the term "web component(s)"

So I'll do the same update here